src/pages/Touringcar/components/MapLayers/MapLayers.test.tsx

```diff
@@ -1,72 +0,0 @@
-import { act, screen, waitFor } from '@testing-library/react'
-import userEvent from '@testing-library/user-event'
-import { RouteIds } from 'routes'
-import delay from 'shared/utils/delay'
-import { getGeneratedPath } from 'shared/utils/path'
-
-import { withApp } from '../../../../../test/utils/withApp'
-
-describe('RoutesLayers', () => {
```

Comment:
Are these tests not relevant anymore?

Reply:
Well, the E2E tests that deal with Amsterdam React Maps/Leaflet cause more trouble than help due to random timeouts and timer issues. So I decided to remove them. We need to find a robust E2E solution at some point.